FromZeroToHero.sln

```diff
@@ -1,15 +1,177 @@
+            }
+        }
+
+        private static void AddElementAtAnyPosition(int[] numbers, int newNumber, int position)
+        {
+            int[] newNumbers = new int[numbers.Length + 1];
+            for (int i = 0; i < numbers.Length + 1; i++)
+            {
+                if (i == position)
+                {
+                    newNumbers[i] = newNumber;
+                }
+                else if (i < position)
+                {
+                    newNumbers[i] = numbers[i];
+                }
+                else
+                {
+                    newNumbers[i] = numbers[i - 1];
+                }
+            }
+            Console.WriteLine();
+            Console.WriteLine("Add element at the the position you want in the array");
+            for (int i = 0; i < numbers.Length + 1; i++)
+            {
+                Console.Write(newNumbers[i] + " ");
+            }
+        }
+        private static void RemoveElementAtStart(int[] numbers)
+        {
+            int[] newArray = new int[numbers.Length - 1];
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                newArray[i] = numbers[i + 1];
+            }
+            Console.WriteLine();
+            Console.WriteLine("Remove first element in array");
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                Console.Write(newArray[i] + " ");
+            }
+        }
+
+        private static void RemoveElementAtEnd(int[] numbers)
+        {
+            int[] newArray = new int[numbers.Length - 1];
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                newArray[i] = numbers[i];
+            }
+            Console.WriteLine();
+            Console.WriteLine("Remove last element in array");
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                Console.Write(newArray[i] + " ");
+            }
+        }
+        private static void RemoveElementAtPositon(int[] numbers, int index)
+        {
+            int[] newArray = new int[numbers.Length - 1];
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                if (i < index)
+                {
+                    newArray[i] = numbers[i];
+                }
+                if (i >= index)
+                {
+                    newArray[i] = numbers[i + 1];
+                }
+
+            }
+            Console.WriteLine();
+            Console.WriteLine("Remove element at any posiion in array");
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                Console.Write(newArray[i] + " ");
+            }
+        }
+        static void LoginInfo(string name, string password)
+        {
+            string myName = "Kin";
+            string pwd = "21Lome";
+            if (myName.Equals(name, StringComparison.OrdinalIgnoreCase) && pwd.Equals(password, StringComparison.OrdinalIgnoreCase))
```

Comment:
This is not correct.  According to the requirements passwords should not be compared with `OrdinalIgnoreCase`.